### ISSUE TITLE
`olive shared-cache` throws exception

### ISSUE THREAD

**Issue by cecheta** [microsoft/Olive#1509]
**Describe the bug**
Using `olive shared-cache` commands throws an exception.

**To Reproduce**
* Run `olive shared-cache --account ACCOUNT --container CONTAINER`
* Error occurs

**Expected behavior**
No error occurs.

**Olive logs**
```
Traceback (most recent call last):
  File "/home/codespace/.python/current/bin/olive", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/olive/cli/launcher.py", line 62, in main
    service.run()
  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/olive/cli/shared_cache.py", line 53, in run
    container_client_factory = AzureContainerClientFactory(
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/python/3.12.1/lib/python3.12/site-packages/olive/common/container_client_factory.py", line 21, in __init__
    credential=get_credentials(**credential_kwargs),
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: get_credentials() got an unexpected keyword argument 'exclude_managed_identity_credential'
```

**Other information**
 - OS: Linux
 - Olive version: main

**Comment by xiaoyu-work** [comment]
Thanks for reporting this. A PR had been created to fix this.

**Comment by jambayk** [comment]
Fixed by #1513
